packages/theming/apple-theme/src/index.ts

```diff
@@ -1,2 +1,3 @@
 export * from './createAppleTheme';
 export * from './createMacOSAliasTokens';
+export { isHighContrastModeEnabled } from './createAppleTheme.macos';
```

Comment:
Is this still being used outside the package?

Reply:
I think so, it gets passed down to `getMacOSAliasTokens` that lives in the theme-tokens package

Reply:
That's the value of the variable though, the variable itself doesn't get imported into `getMacOsAliasTokens.ts`

Reply:
Talked offline, extracting the variable from createAppleTheme and implementing a setter for it did the trick. Thanks!!